doc/api/http.md

```diff
@@ -56,13 +56,13 @@ The HTTP Agent also defaults client requests to using
 socket to become free the socket is closed. This means that Node.js's
 pool has the benefit of keep-alive when under load but still does not
 require developers to manually close the HTTP clients using
-KeepAlive.
+keepAlive.
 
-If you opt into using HTTP KeepAlive, you can create an Agent object
+If you opt into using HTTP keepAlive, you can create an Agent object
```

Comment:
@yorkie I agree that there is some inconsistencies with `Agent` as well. Most of the time when we see capital-A, 'Agent' it's directly preceded by "HTTP".  As in

> The HTTP Agent is used for pooling sockets used in HTTP client requests.

When referred to this way, it seems like it should be referring to "`http.Agent`" (as code, and scoped to the module), instead of "HTTP Agent" (as a non-standard English language description of the object itself).  What do you think?